contrib/scripts/kind.sh

```diff
@@ -75,6 +80,10 @@ image="${4:-${IMAGE:=${default_image}}}"
 kubeproxy_mode="${5:-${KUBEPROXY_MODE:=${default_kubeproxy_mode}}}"
 ipfamily="${6:-${IPFAMILY:=${default_ipfamily}}}"
 apiserver_addr="${7:-${APISERVER_ADDR:=${default_apiserver_addr}}}"
+if [ "${ipv6_only}" = true ]; then
+  ipfamily="ipv6"
```

Comment:
I don't think we need the new flag. The make target can set the IPFAMILY and APISERVER_ADDR.

Reply:
I have to agree for the current change, and I was thinking there would be more related changes to get the external container and DNS working. Maybe I will introduce a flag when things are more complicated.